src/UrlHandlers/TokensApiHandler.php

```diff
@@ -16,10 +17,14 @@ public function __construct(array $childUrlHandlers = [])
     {
         parent::__construct($childUrlHandlers);
 
-        $this->post("/tokens", function($params, WebRequest $request){
+        $this->post("/token", function($params, WebRequest $request){
             return (new TokenResourceAdapter())->post($request->getPayload(), $params, $request);
         })->with(new CredentialsAuthenticationMiddleware());
 
+        $this->post("/token/revoke", function($params, WebRequest $request){
```

Comment:
as i mentioned in your issue, this should really be a `DELETE /token` imo

Reply:
Hey Sam - yes I mentioned that also, although Rob has looked at quite a few auth apis and most use this (non restful) pattern of calling a revoke end point. Might be worth chatting about it again tomorrow as I'm thinking we might be better doing our own thing here rather than matching other oauth api endpoints.

Reply:
Hi @acuthbert I chatted to Rob about this and we have now changed it to be a delete endpoint and the API token is now removed from the db when not being used.

Reply:
I'm going to merge to make sure michael can progress with the team but we can review this and refactor if we need to.